public_html/wp-content/plugins/pattern-directory/includes/pattern-flag-post-type.php

```diff
@@ -145,3 +145,49 @@ function check_flag_threshold( $post_ID, $post, $update ) {
 		) );
 	}
 }
+
+/**
+ * Get a list of post IDs for patterns that have pending flags.
+ *
+ * TODO this isn't used anywhere on the front end, but maybe it should be cached?
+ *
+ * @param array $args Optional. Query args. 'orderby' and/or 'order'.
+ *
+ * @return array
```

Comment:
(Just a suggestion) Do you know what type of items are in the array? I like the `string[]`/`integer[]` format for documenting that.

Reply:
Change in 6a12c28